net/service/handlers.go

```diff
@@ -245,32 +247,48 @@ func (service *OpenBazaarService) handleOfflineRelay(p peer.ID, pmes *pb.Message
 
 func (service *OpenBazaarService) handleOrder(peer peer.ID, pmes *pb.Message, options interface{}) (*pb.Message, error) {
 	offline, _ := options.(bool)
```

Comment:
Can you help me understand why `options` is a bool inside the function but an interface{} outside the function? And maybe why it's named `options` and not `isOffline` or something? If we're overloading this function to accept multiple versions of options, then we should extract the behavior specific to this case and call from within the function. If this is a handler or more generic interface, it should route to other behavior and be as flat as possible. Overloading functions like this make them impossible to test.

Reply:
At the time we had need for different types of data to be passed into there that depended in the specific type of message being handled. But we needed it to conform to the interface. 

Currently on the offline flag is used and only by some of the handlers.